service/Menu.py

```diff
@@ -9,21 +12,34 @@ def __init__(self, handler: MenuHandler):
 		self.handler = handler
 
 	def show(self) -> None:
+		""" Shows the menu that interacts with the user
+
+		:return: None
+		"""
 
 		choice = ''
 		while choice != 'q':
-			print('<1> Search a file to download')
-			print('<2> Search all peers around you')
-			choice = input('Select an option (q to exit):')
+			shell_colors.print_orange('\n-----------------------------------')
+			shell_colors.print_orange('| <1> Search a file to download   |')
+			shell_colors.print_orange('| <2> Search all peers around you |')
+			shell_colors.print_orange('| <3> Add peers manually          |')
+			shell_colors.print_orange('-----------------------------------')
+			shell_colors.print_orange('Select an option (q to exit):')
+			choice = input()
 
 			if choice in {'1', '2'}:
-				if choice == '1':
+				if len(AppData.get_neighbours()) == 0:
+					shell_colors.print_red('\n! You need to have at least one peer to get started !')
```

Comment:
Ottima idea questa, però siccome prima di fare qualsiasi scelta l'utente deve aggiungere almeno un peer, tanto vale chiederglielo almeno una volta all'avvio, e poi lasciargli la possibilità di farlo anche successivamente come in questo caso.